### ISSUE TITLE
Rocket Icon

### ISSUE THREAD

**Issue by JanTrichter** [lucide-icons/lucide#451]
Hello,
I just wanted to open this issue because I couldn't find any rocket icons in the lucide package.
I think it would be a great addition because I've seen it quite often in the past months(mostly for deployment related stuff).

## Icon Request

* Icon name: Rocket
* Use case: Deployment
* _Screenshots_ of similar icons: (to be added later)

Thumbs up if you agree. If we can agree that it is useful I could start creating one or leave it to the community ^^

**Comment by locness3** [comment]
Good one, I think this was already discussed in Feather.

**Comment by maxim-s-barabash** [comment]
Feather issues   [#1062](https://github.com/feathericons/feather/issues/1062)

**Comment by ericfennis** [comment]
![image](https://user-images.githubusercontent.com/11825403/154848816-32867e47-e8c9-4835-a7ee-e20dbf12b754.png)
Created 4 variants, rounded flaps-squared flaps and with-without a window.

Let me know what you think

**Comment by JanTrichter** [comment]
Almost forget about this issue, sorry guys ^^

@ericfennis I think the bottom left is the best since it fits nicely with the smooth and rounded style of lucide/feather. 

I think that both options on the right are a little bit too much and feel a bit "overloaded" with the window.

The top left is also great but I think that the bottom left fits better with the general style of lucide. A side by side comparison with other icons from the package could maybe help to determine what we want to choose but my favourites are the two icons on the left.

**Comment by ericfennis** [comment]
@JanTrichter I agree the bottom ones are a bit softer and looks better with the other icons.
Here the rocket from left bottom with some other icons.
<img width="124" alt="image" src="https://user-images.githubusercontent.com/11825403/157098266-7082e064-8608-4170-a75a-e6f776f14b69.png">
<img width="579" alt="image" src="https://user-images.githubusercontent.com/11825403/157098281-afe093bb-0dd5-4f07-a796-07b1cbf207b2.png">

**Comment by JanTrichter** [comment]
@ericfennis agree, looks really good and fits the feather/lucide style 👍🏽
